packer-plugin-sdk/rpc/init.go

```diff
@@ -1,3 +1,9 @@
+/*
+Package rpc contains the implementation of the remote procedure call code that
+the Packer core uses to communicate with packer plugins. As a plugin maintainer,
+You are unlikely to need to directly import or use this package, but it
+underpins the packer server that all plugins must implement.
+*/
```

Comment:
Suggestion for consistency :
```suggestion
// Package rpc contains the implementation of the remote procedure call code that
// the Packer core uses to communicate with packer plugins. As a plugin maintainer,
// You are unlikely to need to directly import or use this package, but it
// underpins the packer server that all plugins must implement.
```